InfiniteRecharge/src/main/java/frc/robot/RobotMap.java

```diff
@@ -152,6 +205,47 @@ public static SensoredSystem getRobotWranglerSystem() {
         return system;
     }
 
+    public static SensoredSystem getIntakeActuationSystem() {
+        WPI_TalonSRX intakeActuation = new WPI_TalonSRX(kItntakeActuatorCanID);
```

Comment:
may want to change these to be outside of the getter since it will create a whole new object every time we call this... the way the shooter devices get constructed is much better. this can be done in another pr